PocketKit/Sources/PocketGraphTestMocks/PageInfo+Mock.graphql.swift

```diff
@@ -10,10 +10,10 @@ public class PageInfo: MockObject {
   public typealias MockValueCollectionType = Array<Mock<PageInfo>>
 
   public struct MockFields {
-    @Field<String?>("endCursor") public var endCursor
```

Comment:
You need to keep these as optionals for our tests to work

Reply:
@bassrock have we been manually updating these? This is generated.

Reply:
Yea we have.

Reply:
When i raised this with Apollo, i was pushed back on by them cause we are using mocks in an unsupported way.